### PR TITLE
Feat/seo alt sup

### PR DESCRIPTION
**Pull request by LeoDELPON**
SEO Optimization :
- Schema data structure for reviews (Redux, redux-saga), 
- Schema data structure for global review (Redux, redux-saga),
- Schema data structure for BreadCrumb,
- Schema data structure for Addresses,
- Binding internal mesh with review's page,
- Binding All alt props for images,
- Refactoring infrastructure with Hn system,
- Binding internal mesh in provider detail page,
- Refactoring the page for annuaire, annonce and votes for having multiple pages,
- Restructuring the Head component for dynamic schema,

### CODE REVIEW

File: pages/[helpService]/annonce/[searchType]/page/index/[annoncePage].tsx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+        pageNumber: currentPage,
+        numberOfElementsPerPage: PAGE_SIZE,
+      });
+  }, [currentPage]);
+  console.log(props.services);
+  return (
+    <MDPage home>
+      <MDHead description="Annonces de services">
+        <title>Annonces de déménagement</title>
+        <link rel="icon" href="/favicon.ico" />
```
Comment (Tsekko) [inline pages/[helpService]/annonce/[searchType]/page/index/[annoncePage].tsx:74]: Missing canonical link

File: pages/[helpService]/annonce/[searchType]/index.tsx [hunk 2]
```diff
@@ -69,7 +69,7 @@ const AnnuaireDemenagement = (
   console.log(props.services);
   return (
     <MDPage home>
-      <MDHead description="Annuaire d'annonces de déménagement">
+      <MDHead description="Annonces de services">
```
Comment (Tsekko) [inline pages/[helpService]/annonce/[searchType]/index.tsx:72]: Missing canonical link

File: pages/[helpService]/annuaire/[serviceProvider]/[locationId]/[profilId].tsx [hunk 3]
```diff
@@ -46,8 +46,8 @@ const GrosBrasProfil = (props: {
 
   return (
     <MDPage home>
-      <MDHead description="Déménagement profil gros bras">
-        <title>MyDemenageur - Gros bras</title>
+      <MDHead description="Profil de prestataire">
+        <title>MyDemenageur - Profil de {props.name}</title>
```
Comment (Tsekko) [inline pages/[helpService]/annuaire/[serviceProvider]/[locationId]/[profilId].tsx:50]: Missing canonical link

**Review by Tsekko — CHANGES_REQUESTED** [review]

File: pages/profil/password.tsx [hunk 4]
```diff
@@ -33,15 +33,23 @@ const PasswordSection = (props: {
 
   return (
     <MDPage>
-      <MDHead description="Devenir préstataire page">
-        <title>Mot de passe</title>
+      <MDHead description="Page de modification du mot de passe de l'utilisateur">
+        <title>Profil - Mot de passe</title>
+        <link
+          rel="canonical"
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (LeoDELPON) [reply pages/profil/password.tsx:40]: Deleted HEAD token

**Pull request merged**